cssdb.settings.json

```diff
@@ -1178,23 +1178,11 @@
     "specification": "https://www.w3.org/TR/css-fonts-4/#system-ui-def",
     "stage": 2,
     "browser_support": {},
-    "browser_support_overrides": {
-      "safari": {
-        "from": null,
-        "to": "11",
-        "bug": "https://github.com/mdn/browser-compat-data/issues/17337"
-      },
-      "ios_saf": {
-        "from": null,
-        "to": "11",
-        "bug": "https://github.com/mdn/browser-compat-data/issues/17337"
-      }
-    },
     "docs": {
       "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#Syntax"
     },
     "example": "body {\n  font-family: system-ui;\n}",
-    "mdn_path": "css.properties['font-family'].system_ui",
+    "mdn_path": "css.properties['font-family'].system-ui",
```

Comment:
https://github.com/mdn/browser-compat-data/pull/17476/files

Key in `mdn` was changed :

<img width="1087" alt="Screenshot 2022-09-12 at 09 21 26" src="https://user-images.githubusercontent.com/11521496/189595698-ab1b67ec-5348-4876-a3f8-5ee1b2ac4eee.png">